torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -1745,3 +1745,76 @@ def rref_tensor_is_owner(rref_var):
 
         res = rref_tensor_is_owner(rref_var)
         self.assertEqual(res, False)
+
+    @dist_init
+    def test_rpc_async_op(self):
+        @torch.jit.script
+        def two_args_two_kwargs(
+            first_arg,
+            second_arg,
+            first_kwarg=torch.tensor([3, 3]),
+            second_kwarg=torch.tensor([4, 4]),
+        ):
+            return first_arg + second_arg + first_kwarg + second_kwarg
+
+        if self.rank != 0:
+            return
+
+        @torch.jit.script
+        def rpc_async_in_torchscript(dst_worker_name, args, kwargs):
+            # type: (str, Tuple[Tensor, Tensor], Dict[str, Tensor])
```

Comment:
There was a flake8 lint error.

> E999 - Parsing Error. This signifies a parsing error on the line mentioned. Check for Python2/3 syntax differences and if you have type hints make sure they are using e.g. `typing.List` instead of `list`

Then I noticed that we have to mark this file as a python3 only file by adding
```
#!/usr/bin/env python3
```
at the top, to tell the linter that we don't need py2/py3 compatibility lint rules on this file.